.changeset/itchy-cycles-develop.md

```diff
@@ -0,0 +1,8 @@
+---
+'@api3/airnode-deployer': minor
+'@api3/airnode-examples': minor
+'@api3/airnode-node': minor
+'@api3/airnode-validator': minor
+---
+
+Add option to set maximal concurrency for HTTP gateway
```

Comment:
[Very nit](https://www.quora.com/What-are-the-differences-between-%E2%80%9Cmaximum%E2%80%9D-and-%E2%80%9Cmaximal%E2%80%9D-Especially-in-scientific-papers#:~:text=K%20answer%20views-,Originally%20Answered%3A%20What%20is%20the%20difference%20between%20maximum%20and%20maximal,or%20a%20noun%20or%20adverb.): 

```suggestion
Add option to set maximum concurrency for HTTP gateway
```